di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/config/uatuser/IdentityMapper.java

```diff
@@ -47,9 +52,9 @@ public Identity map(Map<String, String> map, int rowNumber) {
 
         Instant dob = Instant.parse(map.get("dob"));
         Instant dateOfEntryOnCtdb = Instant.parse(map.get("dateOfEntryOnCtdb"));
-        DateOfBirth dateOfBirth = new DateOfBirth(dob, dateOfEntryOnCtdb);
+        FindDateOfBirth dateOfBirth = new FindDateOfBirth(dob, dateOfEntryOnCtdb);
 
-        Name name = new Name(map.get("name"), map.get("surname"));
+        FullName name = new FullName(map.get("name"), map.get("surname"));
```

Comment:
I don't think you're using this - what's it for?

Reply:
I have renamed object we had earlier as `Name` to `FullName`. This is again specific to our KBV dataset from which we are picking up the name of the identity.

I wanted `Name` to be used as our main object in the `shared_claims` and hence shuffling it around to keep objects in `shared_claims` as close to as possible